refile.gemspec

```diff
@@ -34,9 +34,8 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "aws-sdk"
   spec.add_development_dependency "rack-test", "~> 0.6.2"
   spec.add_development_dependency "rails", "~> 4.1.8"
-  spec.add_development_dependency "sqlite3"
+  spec.add_development_dependency RUBY_PLATFORM == "java" ? "activerecord-jdbcsqlite3-adapter" : "sqlite3"
```

Comment:
This does not work as you intend it to. This will work fine when developing, but when the gem is generated, the gemspec will depend on the platform it was *built on*, not that it is *run on*. This isn't a huge deal with dev-dependencies, but it's still weird.

Maybe we should just move all dev dependencies into the Gemfile.